docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc

```diff
@@ -43,14 +43,16 @@ Hide on login page::
 If this identity provider should be hidden in login pages when the user is authenticating in the scope of the organization.
 
 Redirect when email domain matches::
-If members should be automatically redirected to the identity provider when their email domain matches the domain set to the identity provider.
+If members should be automatically redirected to the identity provider when their email domain matches the domain set to the identity provider. If the domain is set to `Any`, members whose e-mail matches *any* of the organization domains will be redirected to the identity provider.
```

Comment:
nit: maybe this is better? since the first sentence also explicitly speaks about email doamin?
`members whose e-mail`
->
`members whose email domain`